config/application.rb

```diff
@@ -35,11 +35,11 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :sidekiq
 
-    config.lograge.enabled = true
-    config.lograge.enabled = Lograge::Formatters::Json.new
-    config.log_formatter = proc do |severity, datetime, progname, msg|
-      "time=\"#{datetime.iso8601(3)}\" level=\"#{severity}\" pid=\"#{Process.pid}\" #{msg.to_s.gsub('"', "'")}\n"
-    end
+    # config.lograge.enabled = true
+    # config.lograge.enabled = Lograge::Formatters::Json.new
+    # config.log_formatter = proc do |severity, datetime, progname, msg|
+    #   "time=\"#{datetime.iso8601(3)}\" level=\"#{severity}\" pid=\"#{Process.pid}\" #{msg.to_s.gsub('"', "'")}\n"
+    # end
```

Comment:
@Darshan-Chauhan causing a problem or just hard to read in dev mode? Feel free to move this into production and staging environments, if so.